virtual_rainforest/models/abiotic/abiotic_tools.py

```diff
@@ -40,14 +46,19 @@ def calc_molar_density_air(
             temperature in Kelvin
 
     Returns:
-        molar density of air
+        molar density of air, [kg m-3]
     """
 
     temperature_kelvin = temperature + celsius_to_kelvin
-    molar_density_air = (
-        standard_mole
-        * (temperature_kelvin / atmospheric_pressure)
-        * (celsius_to_kelvin / temperature_kelvin)
+    molar_density_air = DataArray(
+        (
+            standard_mole
+            * (temperature_kelvin / atmospheric_pressure)
+            * (celsius_to_kelvin / temperature_kelvin)
+        ),
+        dims=temperature.dims,
+        coords=temperature.coords,
+        name="molar_density_air",
     )
```

Comment:
The result of all those operations should be a DataArray with the same dimensions and coordinates as the temperature and the atmospheric pressure, so you jus need to rename it, not create a new one:
```suggestion
    molar_density_air =  (
            standard_mole
            * (temperature_kelvin / atmospheric_pressure)
            * (celsius_to_kelvin / temperature_kelvin)
        ).rename("molar_density_air")
```